eZ/Publish/Core/FieldType/Url/Type.php

```diff
@@ -82,7 +82,7 @@ protected function createValueFromInput($inputValue)
      */
     protected function checkValueStructure(BaseValue $value)
     {
-        if (!is_string($value->link)) {
+        if (isset($value->link) && !is_string($value->link)) {
```

Comment:
Are you trying to make it only check type if not null? if so just do `$value->link !== null`, easier to read, which we will do many many times.

Reply:
Edit: I see you just follow example below, but if my assumption is right above, feel free to change both.

Reply:
Don't know if that's the case right now, but ValueObject for thies field type expects two strings (at least in phpdoc), don't know how changing this to allow ints for example would affect rest of the code. any insight on that?

Reply:
Hmm, `isset` in this case is not self-explanatory. I like @andrerom's suggestion: `$value->link !== null` (or `null !== $value->link` if you like yoda style ;) ). And yes, change both please :)

@andrerom WDYT about backporting it to `6.7`?

Reply:
> @andrerom WDYT about backporting it to 6.7?

yes pease, it's a bug fix